### PR TITLE
Update USAspending README with db subset

### PR DESCRIPTION
**Pull request by jdodd2**
Updated docker compose to now only run management commands via command line and updated README with better instructions on how to download and use the database subset we provide on our website

**Description:**
PR to update USAspending README instructions in order to leverage monthly database subset download which gets created every month

**Technical details:**
N/A

**Requirements for PR merge:**

1. [N/A] Unit & integration tests updated
2. [x] API documentation updated
3. [ ] Necessary PR reviewers:
    - [x] Backend
    - [ ] Frontend <OPTIONAL>
    - [ ] Operations <OPTIONAL>
    - [ ] Domain Expert <OPTIONAL>
4. [N/A] Matview impact assessment completed
5. [N/A] Frontend impact assessment completed
6. [N/A] Data validation completed
7. [x] Appropriate Operations ticket(s) created
8. [x] Jira Ticket [OPS-774](https://federal-spending-transparency.atlassian.net/browse/OPS-774):
    - [x] Link to this Pull-Request
    - [ ] Performance evaluation of affected (API | Script | Download)
    - [ ] Before / After data comparison

**Area for explaining above N/A when needed:**
```
USAspending docker-compose.yaml updated to accept any added commands instead of being specific could potentially cause issues if there were dependencies elsewhere, impact will be assessed.
```

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -18,11 +18,11 @@ services:
...
 
-  usaspending-db-migrate:
+  usaspending-manage:
     build: .
     volumes:
      - .:/dockermount
-    command: /bin/sh -c "python -u manage.py migrate"
+    command: /bin/sh -c "python -u manage.py shell"
     environment:
       DATABASE_URL: postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/data_store_api
```
Comment (tony-sappe) [inline docker-compose.yaml]: Should add `ES_HOSTNAME` env var to this app as some management commands need it

File: README.md [hunk 2]
```diff
@@ -52,17 +54,23 @@ See below for basic setup instructions. For help with Docker Compose:
...
 
-	- `docker-compose up usaspending-ref-data` will load essential reference data (agencies, program activity codes, CFDA program data, country codes, and others).
+	- `docker-compose run usaspending-manage python -u manage.py load_reference_data` will load essential reference data (agencies, program activity codes, CFDA program data, country codes, and others).
 
 	- `docker-compose up usaspending-db-sql`, then `docker-compose up usaspending-db-init` will provision the custom materialized views which are required by certain API endpoints.
 
 #### Manual Database Setup
 - `docker-compose.yaml` contains the shell commands necessary to set up the database manually, if you prefer to have a more custom environment.
 
+#### Option 3: Downloading the database or a subset of the database and loading it into PosgreSQL
```
Comment (tony-sappe) [inline README.md]: Can you re-work the header levels? The headers between option 2 and 3 break the structure

File: README.md [hunk 3]
```diff
@@ -28,9 +28,11 @@ Now, navigate to the base file directory where you will store the USAspending re
     $ cd usaspending-api
 
 ## Database Setup
-There are two options for how you want to setup your database in order to run the API.  You can:
-    1. Use your own local postgres database for the API to use.
-    2. Create an empty directory on your localhost where all the database files will persist and use the docker-compose file to bring up a containerized postgres database
+There are three options for how you want to setup your database in order to run the API. You can:
```
Comment (tony-sappe) [inline README.md]:
```suggestion
There are three documented options for setting up a local database in order to run the API:
```

Comment (jdodd2) [reply README.md]: Update made

**Pull request merged**